pepper-apis/studybuilder-cli/studies/pancan/patches/add-additional-diagnosis.conf

```diff
@@ -0,0 +1,241 @@
+{
+  "description": "Adds additional cancer diagnosis to the Genitourinary category (PEPPER-36)",
+  "study": "cmi-pancan",
+
+  "i18n": {
+    "en": {
+      "cancer": {
+        "c_genitourinary_chrcc_renal_cell": "Kidney Cancer / Chromophobe renal cell carcinoma (chRCC)",
+        "c_genitourinary_ccrcc_renal_cell": "Kidney Cancer / Clear-cell renal cell carcinoma (ccRCC)",
+        "c_genitourinary_collecting_duct_carcinoma": "Kidney Cancer / Collecting duct carcinoma",
+        "c_genitourinary_fh_deficient_rcc": "Kidney Cancer / FH deficient RCC",
+        "c_genitourinary_prcc_renal_cell": "Kidney Cancer / Papillary renal cell carcinoma (PRCC)",
+        "c_genitourinary_renal_medullary_carcinoma": "Kidney Cancer / Renal medullary carcinoma",
+        "c_genitourinary_renal_oncocytoma": "Kidney Cancer / Renal oncocytoma",
+        "c_genitourinary_sdha_b_rcc": "Kidney Cancer / SDHA/B RCC",
+        "c_genitourinary_trcc_renal_cell": "Kidney Cancer / Translocation renal cell carcinoma (tRCC)",
+        "c_genitourinary_unclassified_rcc": "Kidney Cancer / Unclassified RCC"
+      }
+    },
+
+    "es": {
+      "cancer": {
+        "c_genitourinary_chrcc_renal_cell": "Cancer de riñon / Carcinoma de células renales cromófobo",
```

Comment:
I believe that it should be **Cáncer de riñón** (note the accents). But do check against documentation provided to you (I did not see it in ticket).